packages/typescript-estree/tests/lib/__snapshots__/parse.test.ts.snap

```diff
@@ -68,18 +68,18 @@ However, that TSConfig does not include this file. Either:
 - Change ESLint's list of included files to not include this file
 - Change that TSConfig to include this file
 - Create a new TSConfig that includes this file and include it in your parserOptions.project
-See the TypeScript ESLint docs for more info: https://typescript-eslint.io/linting/troubleshooting##i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file"
+See the typescript-eslint docs for more info: https://typescript-eslint.io/linting/troubleshooting##i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file"
 `;
 
 exports[`parseAndGenerateServices invalid project error messages throws when non of multiple projects include the file 1`] = `
-"ESLint was configured to run on \`<tsconfigRootDir>/ts/notIncluded0j1.ts\` using \`parserOptions.project\`: 
+"ESLint was configured to run on \`<tsconfigRootDir>/ts/notIncluded0j1.ts\` using \`parserOptions.project\`:
```

Comment:
@JamesHenry https://github.com/typescript-eslint/typescript-eslint/pull/6147#issuecomment-1336135956 found it! Must have been an auto-fix.